### ISSUE TITLE
Install chrome webdriver to server.

### ISSUE THREAD

**Issue by Taz17** [openaustralia/publicwhip#1340]
The chrome webdriver is needed so that we can capture screenshots of the social sharing cards mentioned in issue #1316

**Comment by mlandauer** [comment]
Take a look at https://github.com/openaustralia/infrastructure/. I'd love to see a PR against that repo for any changes required to the infrastructure. I think you should be able to test most of it out locally. The only thing that would be hard to test would be an increase in disk space, so that would be something I probably would need to do.

**Comment by Taz17** [comment]
Since you mentioned a gem that installs chrome driver once the gem is installed. I did some digging, and found that in the [multiple_cianywhere](https://github.com/planningalerts-scrapers/multiple_cianywhere) it uses `capybara` and `selenium-webdriver`. I tried to run it on a laptop I have without the webdriver installed by me and it did not run (driver not found exception)

Turns out morph chromedriver is installed as part of the build process for every docker container. Mentioned in the [documentation](https://morph.io/documentation/scraping_javascript_sites)
